api/mailbox/mailbox.spec.js

```diff
@@ -0,0 +1,52 @@
+const app = require('../../app');
+
+const expect = require('chai').expect;
+const request = require('supertest');
+
+const mailboxDao = require('../../dao/mailbox/');
+
+
+describe('/mailbox api', () => {
```

Comment:
Avoid using Arrow Functions in describe() and it():
http://remarkablemark.org/blog/2016/08/20/arrow-functions-in-mocha/